# Message definition for Battery Data
Header header

uint8 status

string name

uint8 remainingPwr
uint16 remainingTime

float32 current
float32 voltage
uint16 power